src/websocket_service/mod.rs

```diff
@@ -112,6 +119,57 @@ async fn handle_msg(
     Ok(())
 }
 
+async fn resubscribe(
+    database: &Arc<Database>,
+    client: &Arc<relay_client::websocket::Client>,
+) -> Result<()> {
+    info!("Resubscribing to all topics");
+    // Get all topics from db
+    let cursor = database
+        .collection::<LookupEntry>("lookup_table")
+        .find(None, None)
+        .await?;
```

Comment:
Depending on how many topics there are, you may need to set up some pagination?

Reply:
Good idea, will leave it for separate PR thought, as I want to first merge this, as it currently broken.
Made an issue: https://github.com/WalletConnect/cast-server/issues/65